src/main/resources/assets/extratrees/lang/en_US.lang

```diff
@@ -58,8 +58,11 @@ extratrees.block.planks.olive=Olive
 extratrees.block.planks.sweetgum=Sweetgum
 extratrees.block.planks.rosewood=Rosewood
 extratrees.block.planks.gingko=Gingko
-extratrees.block.planks.pinkivory=Pink Ivory
-
+# extratrees.block.planks.pinkivory=Pink Ivory
```

Comment:
It is right key (With a space):
extratrees.block.planks.pink ivory=Pink Ivory
I believe that this line is not needed:
extratrees.block.planks.pinkivory=Pink Ivory
Maybe I'm wrong.